server/plugin/command.go

```diff
@@ -15,37 +15,41 @@ import (
 )
 
 const (
-	featureIssueCreation = "issue_creations"
-	featureIssues        = "issues"
-	featurePulls         = "pulls"
-	featurePullsMerged   = "pulls_merged"
-	featurePullsCreated  = "pulls_created"
-	featurePushes        = "pushes"
-	featureCreates       = "creates"
-	featureDeletes       = "deletes"
-	featureIssueComments = "issue_comments"
-	featurePullReviews   = "pull_reviews"
-	featureStars         = "stars"
-	featureReleases      = "releases"
+	featureIssueCreation      = "issue_creations"
+	featureIssues             = "issues"
+	featurePulls              = "pulls"
+	featurePullsMerged        = "pulls_merged"
+	featurePullsCreated       = "pulls_created"
+	featurePushes             = "pushes"
+	featureCreates            = "creates"
+	featureDeletes            = "deletes"
+	featureIssueComments      = "issue_comments"
+	featurePullReviews        = "pull_reviews"
+	featureStars              = "stars"
+	featureReleases           = "releases"
+	featureDisccusions        = "discussions"
```

Comment:
Typo in variable name.

Reply:
[95e2155](https://github.com/mattermost/mattermost-plugin-github/pull/808/commits/95e2155d254a1fc6ed29cb819f0cd511ddd0a0c5)